bentoml/_internal/server/__init__.py

```diff
@@ -71,20 +74,13 @@ def _start_ngrok_server() -> None:
 
 def serve_production(
     bento_path_or_tag: str,
-    working_dir: Optional[str] = None,
-    port: Optional[int] = None,
+    working_dir: t.Optional[str] = None,
+    app_workers: t.Optional[int] = None,
+    runner_workers: t.Optional[int] = None,
+    port: t.Optional[int] = None,
 ) -> None:
-    if working_dir is not None:
-        working_dir = os.path.abspath(working_dir)
-
     svc = load(bento_path_or_tag, working_dir=working_dir)
 
-    import psutil
```

Comment:
@bojiang we no longer need to limit `serve --production` to POSIX users right?  I think uvicorn have good support for windows

Reply:
Theoretically, yes. I plan to reorganize windows serving when other parts' ready